cache/sampleStoreTimeout.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Just curious, in your testing, did you get a sense of how long this takes with the sampleStore feature on vs. off (with a dataset close to our current prod dataset)?